File: Contest/ICPC 2023/g.py
for run in range(int(input())):
    s= input()
    wicket = 0
    run = 0
    for ele in s:
        if ele in 'Ww':
            wicket+=1
        else:
            run+=(int(ele))
    over = len(s)//6
    ball = len(s)%6
    if run <= 1:
        if wicket<=1:
            if(over == 0):
                print(f'{over}.{ball} Over {run} Run {wicket} Wicket.')
            elif over == 1:
                if ball > 0:
                    print(f'{over}.{ball} Overs {run} Run {wicket} Wicket.')
                else:
                    print(f'{over}.{ball} Over {run} Run {wicket} Wicket.')
            else:
                print(f'{over}.{ball} Overs {run} Run {wicket} Wicket.')
        else:
            if(over == 0):
                print(f'{over}.{ball} Over {run} Run {wicket} Wickets.')
            elif over == 1:
                if ball > 0:
                    print(f'{over}.{ball} Overs {run} Run {wicket} Wickets.')
                else:
                    print(f'{over}.{ball} Over {run} Run {wicket} Wickets.')
            else:
                print(f'{over}.{ball} Overs {run} Run {wicket} Wickets.')
    else:
        if wicket<=1:
            if(over == 0):
                print(f'{over}.{ball} Over {run} Runs {wicket} Wicket.')
            elif over == 1:
                if ball > 0:
                    print(f'{over}.{ball} Overs {run} Runs {wicket} Wicket.')
                else:
                    print(f'{over}.{ball} Over {run} Runs {wicket} Wicket.')
            else:
                print(f'{over}.{ball} Overs {run} Runs {wicket} Wicket.')
        else:
            if(over == 0):
                print(f'{over}.{ball} Over {run} Runs {wicket} Wickets.')
            elif over == 1:
                if ball > 0:
                    print(f'{over}.{ball} Overs {run} Runs {wicket} Wickets.')
                else:
                    print(f'{over}.{ball} Over {run} Runs {wicket} Wickets.')
            else:
                print(f'{over}.{ball} Overs {run} Runs {wicket} Wickets.')
    

File: Contest/ICPC 2023/g.cpp
#include<iostream>
#include<string>
using namespace std;
int main(){
  int t;
  cin >> t;
  for(int j = 0; j<t; j++){
    string s;
    cin >> s;
    long long wicket = 0;
    long long run = 0;
    for(int i =0; i<s.length();i++){
      if(s[i] == 'w' || s[i] == 'W') ++wicket;
      else run += (s[i] - '0');
    }
    int over = s.length()/6;
    int ball = s.length()%6;
    if(run <= 1){
      if(wicket <= 1){
        if(over == 0) cout << over << "." << ball << " Over " << run << " Run " << wicket << " Wicket." << endl;
        else if(over == 1){
          if(ball > 0) cout << over << "." << ball << " Overs " << run << " Run " << wicket << " Wicket." << endl;
          else cout << over << "." << ball << " Over " << run << " Run " << wicket << " Wicket." << endl;
        }else cout << over << "." << ball << " Overs " << run << " Run " << wicket << " Wicket." << endl;
      }else{
        if(over == 0) cout << over << "." << ball << " Over " << run << " Run " << wicket << "Wickets." << endl;
        else if(over == 1){
          if(ball > 0) cout << over << "." << ball << " Overs " << run << " Run " << wicket << " Wickets." << endl;
          else cout << over << "." << ball << " Over " << run << " Run " << wicket << " Wickets." << endl;
        }else cout << over << "." << ball << " Overs " << run << " Run " << wicket << " Wickets." << endl;
      }
    }else{
      if(wicket <= 1){
        if(over == 0) cout << over << "." << ball << " Over " << run << " Runs " << wicket << " Wicket." << endl;
        else if(over == 1){
          if(ball > 0) cout << over << "." << ball << " Overs " << run << " Runs " << wicket << " Wicket." << endl;
          else cout << over << "." << ball << " Over " << run << " Runs " << wicket << " Wicket." << endl;
        }else cout << over << "." << ball << " Overs " << run << " Runs " << wicket << " Wicket." << endl;
      }else{
        if(over == 0) cout << over << "." << ball << " Over " << run << " Runs " << wicket << " Wickets." << endl;
        else if(over == 1){
          if(ball > 0) cout << over << "." << ball << " Overs " << run << " Runs " << wicket << " Wickets." << endl;
          else cout << over << "." << ball << " Over " << run << " Runs " << wicket << " Wickets." << endl;
        }else cout << over << "." << ball << " Overs " << run << " Runs " << wicket << " Wickets." << endl;
      }
    }
  }
  return 0;
}